lib/timeline/component/Group.js

```diff
@@ -498,7 +496,7 @@ Group.prototype._updateSubgroupsSizes = function () {
       })
 
       me.subgroups[subgroup].start = newStart;
-      me.subgroups[subgroup].end = newEnd;
+      me.subgroups[subgroup].end = new Date(newEnd - 1) // -1 to compensate for colliding end to start subgroups;
```

Comment:
Yeah, it can be `newEnd-2` but this is a simple solution for the colliding subgroups. This is the simplest solution to avoid collision of subgroup items. These are by definition a wrapper for items and don't have a real meaning as items. That's why solving their limits by subtracting 1 ms is more correct than fixing the `isColliding` function in `stack.js` file in my opinion.